### DISCUSSION TITLE
How do I get started

### DISCUSSION THREAD

**Discussion by zabirauf** [JuliaPluto/PlutoSliderServer.jl#18] [Q&A] [answered]
Hi,

I love how the computation thinking website has notebooks that are interactive and I'm trying to do something for my own website. I was able to start the server using the instruction from `How to run it, not develop it` but I'm not sure what to do after that. When I visit the URL then I just get **Hi**. How do I ask it to serve the interactive HTML of the notebook?

**Comment by jbrea** [comment]
Have a look at https://github.com/JuliaPluto/PlutoSliderServer.jl/blob/main/test/runtestserver.jl. The settings there work for me.

**Reply by fonsp** [reply]
(This answer is outdated, `runtestserver.jl` is only meant for development of PlutoSliderServer itself, not an example of using it)

**Comment by fonsp** [comment, marked as answer]
Good news! I wrote a big new README! Take a look at the new https://github.com/JuliaPluto/PlutoSliderServer.jl which shows examples of varying complexity. Open a new discussion if you have any questions!
